sdk/storage/Azure.Storage.Files.Shares/src/Models/FileShareRequestConditions.cs

```diff
@@ -0,0 +1,27 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+namespace Azure.Storage.Files.Shares.Models
+{
+    /// <summary>
+    /// Specifies lease access conditions for a file.
+    /// </summary>
+    public class FileShareRequestConditions
```

Comment:
Shouldn't this be `ShareFileRequestConditions` to match all the other API names?

Reply:
(Should fix)

Reply:
fixed.